gallery/src/app/op-gallery/gallery.service.ts

```diff
@@ -12,6 +12,8 @@ export class GalleryService {
 
   private cachedResults: any[] = [];
 
+  sortType: string = 'default';
```

Comment:
add field DEFAULT_SORTING_STATE = 'default' and use it in all places instead of 'default' literal